test/artifact_test.js

```diff
@@ -15,6 +15,15 @@ suite('Artifacts', function() {
   var {Netmask}     = require('netmask');
   var assume        = require('assume');
   var helper        = require('./helper');
+  var fs            = require('fs');
+  var crypto        = require('crypto');
+  var randbytes     = require('randbytes');
+  var tmp           = require('tmp');
+  var remoteS3      = require('remotely-signed-s3');
+  var qs            = require('querystring');
+  var urllib        = require('url');
```

Comment:
Yes, this name is intentional.  There are already lines with `var url` and I don't want mess with those.